Jenkinsfile

```diff
@@ -28,8 +28,15 @@ app.build(application: application, cluster: cluster, deployerVersion: deployerV
   }
 
   app.dockerStage('Container Build') {
-    sh "docker build -t ${appImage} ."
-    sh "docker push ${appImage}"
+    try {
+      github.setImageBuildState(scmVars.GIT_COMMIT, scmVars.GIT_URL, 'PENDING')
```

Comment:
This fails:

```
ERROR: [GitHub Commit Status Setter] Failed to update commit state on GitHub. Ignoring exception [{"message":"No commit found for SHA: f3c81b0970facf9abc8a4663b008eeea55025d46","documentation_url":"https://developer.github.com/v3/repos/statuses/#create-a-status"}]
```

Reply:
![Screen Shot 2019-12-05 at 11 06 33 AM](https://user-images.githubusercontent.com/431299/70242270-4d99c600-174f-11ea-9fcd-d7c42f74c33c.png)

Seems to be working now